### ISSUE TITLE
[Feature Request] Adding a Custome Search Lists to your skin.

### ISSUE THREAD

**Issue by Goldenfreddy0703** [nfm886/skin.cosmic#4]
Hey there, this is a big feature request i wanna request this but it will take some time. Would there be any possible chance to add a Custome Search Lists to your skin in the future. Your skin is really awesome and lightweight and having a Custome Search Lists would be a great idea for your skin so it can be like your using auramod or artic horizon but light weight. This probably will take some time and if you want, you can put this request on hold if you need to but again, great skin and hope to use it more in the future. 

![image](https://user-images.githubusercontent.com/62456796/128562195-d4728fe2-b0c4-4990-a8d2-fd66fa2922ed.png)
![image](https://user-images.githubusercontent.com/62456796/128562299-3c31d0e2-b325-4d30-a0fa-3c9e7ffbc6a3.png)

**Comment by icpryde** [comment]
Ya custom search lists is a must like arctic horizon and ressurection both have it

**Comment by icpryde** [comment]
To build off this custom search lists on arctic horizon and Resurrection

 it looks like this:

![Screen Shot 2021-08-11 at 4 13 48 PM](https://user-images.githubusercontent.com/29389746/129100777-1a9224af-5d4a-4c07-9915-a35311ded632.png)

![Screen Shot 2021-08-11 at 4 13 56 PM](https://user-images.githubusercontent.com/29389746/129100820-4d521eba-4812-4734-ae82-ac5328a41f69.png)

That way for example when you can have multiple things search in the same field like this where you can have it populate both movies and tv shows (or whatever you want really).

![Screen Shot 2021-08-11 at 4 48 46 PM](https://user-images.githubusercontent.com/29389746/129101254-668052cf-400e-42a1-8fd6-db38c5946bfe.png)

**Comment by nfm886** [comment]
Hi!
Just added this feature. Would be nice if you can test it. You can do it by downloading this zip file: https://github.com/nfm886/skin.cosmic/archive/refs/heads/master.zip
The implementation is similar to that one from Arctic Horizon skin. Even I noticed the same bugs ^^
Probably I will push an update in about 1-2 hours.

**Comment by icpryde** [comment]
wow @nfm886 you are freaking fast LMFAO I was just coming here to see if any updates and sure enough you did haha ok ill test it right now for you!

**Comment by nfm886** [comment]
Yeah, please test. From what I tested, bugs may appear here and there. But I noticed them also AH skin so I assume something was with my setup. Will test it again after fresh install.

**Comment by icpryde** [comment]
![Screen Shot 2021-08-24 at 2 34 14 PM](https://user-images.githubusercontent.com/29389746/130671044-eece1f50-a1ae-4c30-8653-9b8581f691bb.png)

![Screen Shot 2021-08-24 at 2 34 24 PM](https://user-images.githubusercontent.com/29389746/130671054-87e4179c-7036-4efa-b0df-30c208dde5aa.png)

It seems to be working, just set it up the same way i do on AH and Ressurection and it does what i expect it to do, the layout seems to be a little off as you can see from the screen shot i can only see 1 row at a time of text and i guess thats because of the rest of the layout and hte skin is following that format?

Thats not such a big deal but you can see some text overlapping other text if that could just be adjusted i think it would be good to go. @nfm886

**Comment by icpryde** [comment]
on further testing, the text over lapping text seems to be on that 1 specific one that has no poster saying "seren" so maybe thats not even a skin issue but just being a random pokemon thing in general cause all the other ones are fine once i move over to the other ones.

![Screen Shot 2021-08-24 at 2 37 00 PM](https://user-images.githubusercontent.com/29389746/130671387-84dba2ba-2902-4255-8c51-c4fec540590a.png)

**Comment by nfm886** [comment]
Thanks for your testing!
Tomorrow I will tweak layout a little bit.
I prefer it to be this way instead of only 2 rows of posters. I will take a closer look on this text overlapping.
Again, thanks and I think I can push an update :)

**Comment by rizreflects** [comment]
> ![Screen Shot 2021-08-24 at 2 34 14 PM](https://user-images.githubusercontent.com/29389746/130671044-eece1f50-a1ae-4c30-8653-9b8581f691bb.png)
> 
> ![Screen Shot 2021-08-24 at 2 34 24 PM](https://user-images.githubusercontent.com/29389746/130671054-87e4179c-7036-4efa-b0df-30c208dde5aa.png)
> 
> It seems to be working, just set it up the same way i do on AH and Ressurection and it does what i expect it to do, the layout seems to be a little off as you can see from the screen shot i can only see 1 row at a time of text and i guess thats because of the rest of the layout and hte skin is following that format?
> 
> Thats not such a big deal but you can see some text overlapping other text if that could just be adjusted i think it would be good to go. @nfm886 
> 

If you wouldn't mind, could you explain how you got this working? For movies, I tried settings the action to seren > search > search movies > new movies search (keyboard pops up & I click done) > create menu item here

Is that path correct? It doesn't seem to be working for me.. any help would be appreciated.

**Comment by icpryde** [comment]
@nfm886 tiny little bug I noticed when i go search and than back out of search it leaves behind a Trakt rating 8.0 and its always 8.0 so it seems like a tiny bug of code somewhere. It doesn't affect anything its just a visual bug.

![Screen Shot 2021-08-24 at 3 56 48 PM](https://user-images.githubusercontent.com/29389746/130681541-be78ed21-4f97-405d-ac56-3a196f800aaa.png)

**Comment by icpryde** [comment]
LOL sorry, I was incorrect it doesn't change but the bug it still the same.

![Screen Shot 2021-08-24 at 4 03 07 PM](https://user-images.githubusercontent.com/29389746/130682263-1dccd388-f278-472f-a92f-258ee1680ee6.png)
